doc/source/installation/digital_alliance.rst

```diff
@@ -200,13 +142,12 @@ Simulations are sent to the scheduler via batch scripts. Visit the Digital Resea
 
   #!/bin/bash
   #SBATCH --account=$yourgroupaccount
-  #SBATCH --ntasks-per-node=$X #number of parallel tasks (as in mpirun -np X)
-  #SBATCH --nodes=1 #number of whole nodes used (each with up to 40 tasks-per-node)
+  #SBATCH --ntasks-per-node=$X #number of parallel tasks per node. When using a full node, this should correspond to the number of cores available on the node. 
```

Comment:
```suggestion
  #SBATCH --ntasks-per-node=$X #number of parallel tasks per node.
```

Reply:
I would put the rest of the comment (`When using a full node, this should correspond to the number of cores available on the node.`) after the code block because like this you have to scroll in the block to see the complete comment.